vuepress/docs/next/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebarDepth: 2
+redirectFrom: /next/tutorials/ecr/tutorials/create-ecr-bundle-from-git.html
+---
+# Build and Publish a Simple Bundle
+
+## Overview
+This tutorial shows you how to create a simple Entando bundle and deploy it into the Entando Component Repository. This involves manually defining a bundle with a single widget, checking your bundle artifacts into git, apply the Entando bundle custom resource to Kubernetes, and then installing the bundle into your application. 
+
+## Prerequisites
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) tool to verify you have the necessary prerequisites in place (e.g. Java, npm, git, etc.). You will also need your git credentials, an available git repository, and an Entando instance. Some of the following steps can be simplified by using the `ent prj` command and its publication system (pbs) convenience methods. The manual steps are also provided in those cases.
+
+## Create the project folder 
+We'll create a parent folder and then a child bundle folder. In a project generated by the Entando Component Generator the microservice and micro frontend source files also live under the parent folder.
+
+``` sh
+mkdir -p example-bundle/bundle; cd example-bundle/bundle 
+```
+## Add a simple widget
+
+Create a widget directory and descriptor file:
+``` sh
+mkdir widgets
+nano widgets/example-widget.yaml
+```
+
+Now let’s populate the `example-widget.yaml` metadata with some content:
+``` yaml
+code: example-widget
+titles:
+   en: Example Widget
+   it: Widget d'esempio
+group: free
+customUi: <h2>Hi from Example Widget</h2>
+```
+
+## Create the bundle descriptor
+
+The `descriptor.yaml` is the main file processed by the Entando Component Repository and describes all of the components in the bundle. 
+``` sh
+nano descriptor.yaml
+```
+Add the following content to the file. You can use your preferred editor but make sure the YAML indentation matches what you see here.
+``` yaml
+code: example-bundle
+description: This is an example of an Entando bundle
+components:
+  widgets:
+    - widgets/example-widget.yaml
+```
+The component descriptor file name and location is arbitrary since the bundle descriptor explicitly points to it. The usual convention is to group components by type.
+
+## Publish the bundle
+
+You can publish a bundle using the CLI or you can perform the steps by hand.
+### CLI steps
+1. From the project directory, initialize npm and accept all of the defaults. Note: this step is unnecessary for a full Project since the Entando Blueprint handles it. 
```

Comment:
Are we using capital Project to refer to "Entando Project".

Reply:
My bad. I went back and forth on whether to capitalize Entando Project and Entando Bundle and finally decided to not capitalize either, except in page or section titles. I think that's more consistent with other documentation we have (including CLI) so I'll make it consistent in the next commit.